src/components/NavDrawer.tsx

```diff
@@ -232,7 +240,7 @@ export const NavDrawer: React.FC<Props> = ({ children, logout, profile }) => {
               >
                 Cancel
               </Button>
-              <Button onClick={() => logout()} color='secondary'>
+              <Button onClick={clearStorage} color='secondary'>
```

Comment:
@binduak can we rename `clearStorage` to maybe `clearNotificationStorageAndLogout`?